test/test_suite.py

```diff
@@ -116,6 +116,46 @@ def _test_get_table_oid(self, table_name, schema=None):
         oid = insp.get_table_oid(table_name, schema)
         self.assert_(isinstance(oid, int))
 
+    @testing.requires.foreign_key_constraint_option_reflection
+    @testing.provide_metadata
+    def test_get_foreign_key_options(self):
```

Comment:
Could you please move the motivation into the code as comment?

Reply:
Did so. Also mentioned the reason for overwriting table definitions to be used for temporary table tests